@shared/api/internal.ts

```diff
@@ -24,26 +24,14 @@ export const createAccount = async (
   return { publicKey };
 };
 
-export const loadAccount = async (): Promise<{
+export const loadAccount = (): Promise<{
   hasPrivateKey: boolean;
   publicKey: string;
   applicationState: APPLICATION_STATE;
-}> => {
-  let response = {
-    hasPrivateKey: false,
-    publicKey: "",
-    applicationState: APPLICATION_STATE.APPLICATION_STARTED,
-  };
-
-  try {
-    response = await sendMessageToBackground({
-      type: SERVICE_TYPES.LOAD_ACCOUNT,
-    });
-  } catch (e) {
-    console.error(e);
-  }
-  return response;
-};
+}> =>
+  sendMessageToBackground({
+    type: SERVICE_TYPES.LOAD_ACCOUNT,
+  });
```

Comment:
As I started modifying this function to allow the errors to propagate, I realized there's always going to be a response (because `sendMessageToBackground` will either resolve or reject one) so the default wasn't useful, and any `catch` was just going to re-throw the error it got, so it could be removed